frontend/src/main/scala/bloop/engine/tasks/TestTask.scala

```diff
@@ -41,6 +41,20 @@ final case class TestFrameworkWithClasses(
 object TestTask {
   implicit private val logContext: DebugFilter = DebugFilter.Test
 
+  /**
+   * Does the given project represent a test project? Test project are projects tagged with either
+   * `test` or `integration-test`.
+   *
+   * See [[Tag.Test]], [[Tag.IntegrationTest]]
+   *
+   * @param project The project to test
+   * @return true if `project` is a test project, false otherwise.
+   */
+  def isTestProject(project: Project): Boolean = {
+    project.tags.contains(Tag.Test) ||
```

Comment:
The tests seem to be failing, maybe they don't have the tags set properly?

Reply:
That was exactly the problem!